sdk/monitor/Azure.Monitor.Query/tests/MetricsQueryClientTests.cs

```diff
@@ -60,5 +63,45 @@ public void ExposesPublicEndpoint()
             var client = new MetricsQueryClient(new Uri("https://management.azure.gov"), new MockCredential(), new MetricsQueryClientOptions());
             Assert.AreEqual(new Uri("https://management.azure.gov"), client.Endpoint);
         }
+
+        [Test]
+        public void CanGetMetricQueryResult()
+        {
+            MetricTimeSeriesElement metricTimeSeriesElement = new Models.MetricTimeSeriesElement();
+            Assert.IsNotNull(metricTimeSeriesElement);
+            Assert.AreEqual(0, metricTimeSeriesElement.Metadata.Count);
+            Assert.AreEqual(0, metricTimeSeriesElement.Values.Count);
+            IEnumerable<MetricTimeSeriesElement> metricTimeSeriesElements = new[] { metricTimeSeriesElement };
+
+            MetricUnit metricUnit = new MetricUnit("test");
+            Assert.IsNotNull(metricUnit);
+            Assert.AreEqual("test", metricUnit.ToString());
+
+            MetricResult metricResult = new Models.MetricResult("https://management.azure.gov", "type", new LocalizableString("name"), metricUnit, metricTimeSeriesElements);
```

Comment:
Is this MetricResult constructor public?

Reply:
yes

Reply:
Looks like it's internal: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Query/src/Generated/Models/MetricResult.cs